README.md

```diff
@@ -8,6 +8,15 @@ This demo showcases how you can perform the following actions :
 
 - Request Data
 
+
+1. Schema for this send and request data
+
+- [Coupon](https://common.schemas.verida.io/health/pathology/tests/covid19/pcr/v0.1.0/schema.json)
```

Comment:
Coupon title but URL of the covid schema

Reply:
resolved